TeamCode/src/main/java/org/firstinspires/ftc/teamcode/autonomous/AutonomousOpMode.java

```diff
@@ -139,21 +145,49 @@ void driveWithEncoders(double revolutions, double maxPower) throws InterruptedEx
             will be set to maxPower, but when it gets within 1.0 revolutions, the power
             will be ramped down to minPower
         */
-        Ramp ramp = new ExponentialRamp(0.01, minPower, 1.0, maxPower);
+        Ramp ramp = new ExponentialRamp(driveSystem.revolutionsToTicks(0.01), minPower,
```

Comment:
ramp is passed to adjustPower() which now requires the domain (i.e. x1 and x2) to be specified in ticks.